### DISCUSSION TITLE
Is it not possible to stop the noisy cooling fan?

### DISCUSSION THREAD

**Discussion by gitfve** [mriscoc/Ender3V2S1#823] [General]
Ender3 S1 Pro Using latest  "Ender3S1-F4-UBL-20230312.bin" work nice.

I wonder where in the code the noisy FAN (NOT the filament cooler) is controlled. It start when the nozzle is hot, and stop when nozzle is cooled down.

In the "Configuration.h" line 412  the "Power Supply Control" "defines start. 
I see the line:
`//#define AUTO_POWER_E_TEMP 50 // (°C) PSU on if any extruder is over this temperature`

It is comment out in all config, is it possible to stop the noisy cooling fan with some fw change?

**Comment by mriscoc** [comment]
Check
https://github.com/mriscoc/Ender3V2S1/blob/326c8903ad432c6a178a05c0c74ebeb5e2ac6a35/Marlin/Configuration_adv.h#L660

**Comment by gitfve** [comment]
Thanks for that.
-Wonder if there are any problem to set PWM lower ex, 200 or so. AND move the limit for the temp up to 90 dC.
-The fan is so noisy because the air pressure is high caused by the mechanical design.
--Lowering the PWM to reduce the noise and again stopping earlier will be nice, and no problem i think. Normally the nozzle is >180dC 

*So why not set the PWN a little lower ex 200 and raise the temperature to ex 90dC???

**Reply by mriscoc** [reply]
Lowering the air flow of the fan could cause heat creep.
